cuttle-cards/cuttle
# Feature #1132 Navbar logo link to home

## Issue number

Relevant [#1132](https://github.com/cuttle-cards/cuttle/issues/1132)
- Resolves #1132 

## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [x] Have tests been added to cover any new features or fixes?
  - [x] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
Implementation Details:
- Added router-link wrapper with `display: contents` around logo to maintain existing formatting while enabling navigation to home ('/')

Test Coverage:
- Added tests for authenticated users 
- Tests both desktop (1920x1080) and mobile (iPhone-8) viewports
- Verifies clicking logo navigates from /stats to home (/)

To Test:
1. Run `npm run e2e:gui`
2. Select `navigation.spec.js`
3. Verify logo navigation tests pass in both viewports
